### PR TITLE
The line causing the exception has been removed

### PR DESCRIPTION
**Pull request by ljelen**
related to #183

### CODE REVIEW

**Comment by grzanka** [comment]
It looks that [flake8](http://flake8.pycqa.org/en/latest/index.html) complains about [PEP8](https://www.python.org/dev/peps/pep-0008/) problems:

image

https://travis-ci.org/pytrip/pytripgui/jobs/518008348#L784

**Comment by grzanka** [comment]
Appveyor seems more problematic - current pyinstaller 3.4 (see https://pypi.org/project/PyInstaller/#history) seems not work with setuptools 28.x (see https://pypi.org/project/setuptools/#history)

See similar issue here: https://github.com/pyinstaller/pyinstaller/issues/3806#issuecomment-481626352

Few issues:
 - why do we use so old setuptools "tox 3.8.6 has requirement setuptools>=30.0.0, but you'll have setuptools 28.8.0 which is incompatible." in https://ci.appveyor.com/project/grzanka/pytripgui/builds/23724960/job/jnj4h9xjypoy0x32#L86
 - check if upgrading setuptools helps
 - check if using development version of pyinstaller works (i.e. installing it via pip from git master branch)

File: appveyor.yml [hunk 1]
```diff
@@ -43,6 +43,7 @@ install:
 # install usual requirements
   - pip install --upgrade -r requirements.txt
   - pip install --upgrade -r tests\\requirements-test.txt
+  - pip install setuptools>=30.0.0
```
Comment (grzanka) [inline appveyor.yml]: Try installing setuptools before `requirements.txt`

Comment (ljelen) [reply appveyor.yml]: ill try

File: appveyor.yml [hunk 2]
```diff
@@ -41,6 +41,7 @@ install:
 # check pip version
   - pip -V
 # install usual requirements
+  - pip install "setuptools>=30.0.0"
```
Comment (grzanka) [inline appveyor.yml]: Add a comment mentioning why this line was added. Include link to the issue.

File: pytripgui/controller/main_cont.py [hunk 3]
```diff
@@ -139,8 +139,8 @@ def open_dicom(self, ddir):
 
         # TODO: RTplan data
 
-        # add cube to the treeview<s
-        self.tree.update_tree()
+        # add cube to the treeviews
+        # self.tree.update_tree()
```
Comment (grzanka) [inline pytripgui/controller/main_cont.py]: Add a comment explaining why the line was commented out.

Comment (ljelen) [reply pytripgui/controller/main_cont.py]: ok

**Comment by grzanka** [comment]
@ljelen please squash merge once all checks are ok

**Pull request merged**